### PR TITLE
also sync noarch rpm

### PR DESCRIPTION
**Pull request by Ximinhan**
test job: https://art-jenkins.apps.prod-stable-spoke1-dc-iad2.itup.redhat.com/job/aos-cd-builds/job/build%252Fsync-for-ci/4829/console

### CODE REVIEW

File: doozer/doozerlib/cli/__main__.py [hunk 1]
```diff
@@ -280,6 +280,8 @@ def beta_reposync(runtime, output, cachedir, arch, repo_type, names, dry_run):
                 continue
 
             color_print('Syncing repo {}'.format(repo.name), 'blue')
+            if arch == "x86_64":
```
Comment (vfreex) [inline doozer/doozerlib/cli/__main__.py:283]: Could you explain why noarch rpms should only be included in x86_64 repos? I think repos for all arches should include noarch rpms as well.

File: doozer/doozerlib/cli/__main__.py [hunk 2]
```diff
@@ -280,6 +280,8 @@ def beta_reposync(runtime, output, cachedir, arch, repo_type, names, dry_run):
...
+                arch = arch + ' --arch noarch '
```
Comment (vfreex) [inline doozer/doozerlib/cli/__main__.py:284]: The naming of the variable will be misleading: It is named "arch", but it is actually part of command line options.

File: doozer/doozerlib/cli/__main__.py (hunk 1 above)
Comment (Ximinhan) [reply doozer/doozerlib/cli/__main__.py]: updated

**Review by joepvd — APPROVED** [review]
Awesome!
/lgtm

**Pull request merged**